widgets/secretProviders/src/SecretProvidersTable.consts.ts

```diff
@@ -1,9 +1,14 @@
 import type { SecretProvidersWidget } from './widget.types';
 
-export const tableRefreshEvent = 'tokens:refresh';
+export const tableRefreshEvent = 'secretProviders:refresh';
 export const dataSortingKeys: Record<string, keyof SecretProvidersWidget.DataSortingKeys> = {
     name: 'name',
     type: 'type',
     createdAt: 'created_at',
     updatedAt: 'updated_at'
 } as const;
+const translationSecretProvider = Stage.Utils.getT('widgets.secretProviders');
+
+export const translateSecretProviders = (key: string, params?: Record<string, any>) => {
+    return translationSecretProvider(key, params);
+};
```

Comment:
As we are redeclaring and passing again parameters to the `translationSecretProvider` function, could we use it directly then?

```suggestion
export const translateSecretProviders = Stage.Utils.getT('widgets.secretProviders');
```

Or do you see some advantages of creating additional wrapper for that function? 🤔

Reply:
Everyday we learn something new 😁